### PR TITLE
Integrated Travis CI

### PR DESCRIPTION
**Pull request by SachinRammoorthy**
Integrated travis and updated README to show build status.

(If it has not been done yet, kindly enable Travis for this repo on the site)

screen shot 2017-12-22 at 11 51 28 pm
screen shot 2017-12-22 at 11 51 18 pm

### CODE REVIEW

**Comment by SachinRammoorthy** [comment]
You can find the build status and travis CI integrated [here](https://github.com/SachinRammoorthy/notes) in my fork of this repo.

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+language: node_js
+
+sudo: enabled
+
+script:
+  - echo "Hello! This is Travis"
```
Comment (m-murad) [inline .travis.yml]: Add a blank line at the end.

File: README.md [hunk 2]
```diff
@@ -1,5 +1,9 @@
 # Notes
 
+[![Build Status](https://travis-ci.org/jboss-outreach/notes.svg?branch=master)](https://travis-ci.org/jboss-outreach/notes)
+
+
+
 A node application to manage notes
 
 ## Setting up the project
```
Comment (yashovardhanagrawal) [inline README.md]: Add a blank line here too, remember always at the end of files add a blank line. You may read this for more information... Also, if you can please squash your commits
https://stackoverflow.com/questions/2287967/why-is-it-recommended-to-have-empty-line-in-the-end-of-file

Comment (SachinRammoorthy) [reply README.md]: @yashovardhanagrawal there is already a new line at the end...I think you missed the last part of the file.

Comment (yashovardhanagrawal) [reply README.md]: oh yeah sorry my fault

**Pull request merged**